crates/context_aware_config/src/api/config_versions/handlers.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{
+    db::{models::ConfigVersion, schema::config_versions},
+    types::{PaginatedResponse, QueryFilters},
+};
+
+extern crate base64;
+use service_utils::service::types::DbConnection;
+
+use superposition_types::result as superposition;
+
+use actix_web::{
+    get,
+    web::{Json, Query},
+    Scope,
+};
+use diesel::{ExpressionMethods, QueryDsl, RunQueryDsl};
+
+pub fn endpoints() -> Scope {
+    Scope::new("").service(get)
+}
+
+#[get("")]
+async fn get(
+    db_conn: DbConnection,
+    filters: Query<QueryFilters>,
+) -> superposition::Result<Json<PaginatedResponse<ConfigVersion>>> {
```

Comment:
Do we need the Json wrapper struct? Can we implement something on PaginatedResponse<T> to avoid this?